base/tuple.jl

```diff
@@ -16,8 +16,15 @@ NTuple
 
 # convenience function for extracting N from a Tuple (if defined)
 # else return `nothing` for anything else given (such as Vararg or other non-sized Union)
-_counttuple(::Type{<:NTuple{N,Any}}) where {N} = N
-_counttuple(::Type) = nothing
+
+"""arity(::Type{<:NTuple})
+Function
+Returns the number of elements in the tuple type.
+Returns nothing for anything else.
+"""
```

Comment:
This is parsed as markdown, and the formatting is whitespace sensitive. We want the function signature to be formatted as `code`, so either put it in backticks (``` ` ```) or put four spaces in front. The convention as described in the docs, here, is to put four spaces in the front:
https://docs.julialang.org/en/v1/manual/documentation/#Documentation

Can you please edit this to follow the guidelines there? :) thanks!

Also, I'd probably change the type in the docstring example to `::Type{<:Tuple})`, since `Tuple` and `NTuple{N,Any} where N` are equivalent, but `Tuple` is clearer:
```julia
julia> Tuple == NTuple{N,Any} where N
true
```

```suggestion
"""
    arity(::Type{<:Tuple})
Function
Returns the number of elements in the tuple type.
Returns nothing for anything else.
"""
```

Reply:
Docstring formatted.